libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Schemas/Actions/Microsoft.ReplaceDialog.schema

```diff
@@ -18,6 +21,12 @@
                 "type": "string",
                 "title": "Options"
             }
+        },
+        "includeActivity": {
+            "type": "boolean",
+            "title": "Include Activity",
+            "description": "Set to true if you want the new dialog to process the activity.",
```

Comment:
Suggested description text to keep this in line with rest of the tone of voice.

 "When set to true, dialog that is called can process the current activity."